### ISSUE TITLE
I can't generate artemis same input.

### ISSUE THREAD

**Issue by trancongthanh1992** [comigor/artemis#194]
**Bug description**
Please help me for issue error with same input ?

**Specs**
Artemis version: [e.g. 5.1.0]
<details>
<summary>build.yaml:</summary>
  
```yaml
targets:
  $default:
    sources:
      - lib/**
      - lib/graphql/**
      - lib/example.schema.graphql
    builders:
      artemis:
        options:
          fragments_glob: lib/graphql/fragments/**.fragment.graphql
          schema_mapping:

            # Error Same Input
            - schema: lib/example.schema.graphql
              queries_glob: lib/graphql/mutations/vidiSubmitDisplayTracking.graphql
              output: lib/model/graphql/vidiSubmitDisplayTracking.dart
              add_query_prefix: true

```
</details>
<details>
<summary>Artemis output:</summary>
  
```bash
# Please paste the output of
$ flutter pub run build_runner build --verbose
#or
$ pub run build_runner build --verbose


pub finished with exit code 1

Normal log
[INFO] Generating build script...
[INFO] Generating build script completed, took 347ms

[WARNING] The package `flutter_b2b` does not include some required sources in any of its targets (see their build.yaml file).
The missing sources are:
  - $package$
[INFO] Initializing inputs
[INFO] Reading cached asset graph...
[INFO] Reading cached asset graph completed, took 137ms

[INFO] Checking for updates since last build...
[INFO] Checking for updates since last build completed, took 698ms

[INFO] Running build...
[INFO] Running build completed, took 19ms

[INFO] Caching finalized dependency graph...
[INFO] Caching finalized dependency graph completed, took 82ms

[SEVERE] artemis:artemis on lib/$lib$ (cached):

Exception: Two classes were generated with the same name `VidiSubmitDisplayTracking$VidiDisplayTrackingInput$ImageInput`!
You may want to:
- Make queries_glob stricter, to gather less .graphql files on a single output
- Use alias on one of the places a field of type `VidiSubmitDisplayTracking$VidiDisplayTrackingInput$ImageInput` is requested
- File a bug on artemis (https://is.gd/YLSfC2)
[SEVERE] Failed after 133ms

Verbose log
#0      throwToolExit (package:flutter_tools/src/base/common.dart:14:3)
#1      _DefaultPub.interactively (package:flutter_tools/src/dart/pub.dart:338:7)
<asynchronous suspension>
#2      PackagesForwardCommand.runCommand (package:flutter_tools/src/commands/packages.dart:239:15)
#3      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:723:18)
#4      _rootRunUnary (dart:async/zone.dart:1192:38)
#5      _CustomZone.runUnary (dart:async/zone.dart:1085:19)
#6      _FutureListener.handleValue (dart:async/future_impl.dart:141:18)
#7      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:682:45)
#8      Future._propagateToListeners (dart:async/future_impl.dart:711:32)
#9      Future._completeWithValue (dart:async/future_impl.dart:526:5)
#10     Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:556:7)
#11     _rootRun (dart:async/zone.dart:1184:13)
#12     _CustomZone.run (dart:async/zone.dart:1077:19)
#13     _CustomZone.runGuarded (dart:async/zone.dart:979:7)
#14     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1019:23)
#15     _microtaskLoop (dart:async/schedule_microtask.dart:43:21)
#16     _startMicrotaskLoop (dart:async/schedule_microtask.dart:52:5)
#17     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
#18     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:169:5)
```
</details>
<details>
<summary>GraphQL schema:</summary>

```graphql
# If possible, please paste your GraphQL schema file,
# or a minimum reproducible schema of the bug.
input VidiDisplayTrackingInput {
  code: String
  latitude: Float
  longitude: Float
  deviceID: String
  displayProductMedias: [ImageInput]
  posmMedias: [ImageInput]
}

type Mutation {
  vidiSubmitDisplayTracking(input: VidiDisplayTrackingInput): VidiUpdateResultData
}
```
</details>
<details>
<summary>GraphQL query:</summary>

```graphql
# If possible, please paste your GraphQL query file,
# or a minimum reproducible query of the bug.
input VidiDisplayTrackingInput {
  code: String
  latitude: Float
  longitude: Float
  deviceID: String
  displayProductMedias: [ImageInput] ### Error
  posmMedias: [ImageInput] ### Error
}
```
</details>

**Comment by vasilich6107** [comment]
Please use the latest beta 6.6.4-beta.1

**Comment by trancongthanh1992** [comment]
> Please use the latest beta version - 6.6.4-beta.1

Thanks @vasilich6107 . I'm going to try it.

**Comment by vasilich6107** [comment]
@trancongthanh1992 any updates?

**Comment by vasilich6107** [comment]
Closing this issue. Duplication bug was fixed in `v6.0.10-beta.1`
